airbyte-integrations/connectors/destination-s3/src/test/resources/parquet/json_schema_converter/json_conversion_test_cases.json

```diff
@@ -1216,5 +1168,226 @@
       "identifier": ["151", "152", "true", "{\"id\":153}"],
       "_airbyte_additional_properties": null
     }
+  },
```

Comment:
The above case is trying to test objects with same names. However, it had lots of irrelevant fields. Those fields have been removed to make the test case easier to read.